products/dev_alternate_products.md

```diff
@@ -0,0 +1,95 @@
+To create a new Gradle plugin project, follow the tutorial on the [Getting Started with Gradle](/tutorials/build_system/prerequisites.md) page.
+The tutorial produces a skeleton Gradle project suitable to use as a starting point.
+Some minor modifications are needed, as described below, and on the individual product pages in Part VIII.
+
+Understanding the relationship between build numbers is critical when developing a plugin to target an IntelliJ Platform-based IDE:
+* _targetIDE_ is the (version-specific) IntelliJ Platform-based IDE in which the plugin is intended to run, such as PyCharm.
+  The path in Gradle plugin attribute [`runIde.ideaDirectory`](/tutorials/build_system/gradle_guide.md#running-against-alternate-versions-and-types-of-intellij-platform-based-ides) is set to the installed location of _targetIDE_.
+* _baseIntelliJPlatformVersion_ is the (version-specific) IntelliJ Platform used in the build of the _targetIDE_.
+  The IntelliJ Platform is defined by a specific build of the IntelliJ IDEA Community Edition.
+  The Gradle plugin attribute [`intellij.version`](/tutorials/build_system/gradle_guide.md#intellij-platform-configuration) is set to be _baseIntelliJPlatformVersion_.
+
+For API compatibility, the IntelliJ Platform version used in the _targetIDE_ dictates the _baseIntelliJPlatformVersion_ used for developing a plugin. 
+
+### Matching Versions of the IntelliJ Platform with the Target IDE Version
+The _baseIntelliJPlatformVersion_ used in the _targetIDE_ may not be readily apparent, depending on the product.
+See the individual product pages in Part VIII for exceptions.
+
+To find the version of the IntelliJ Platform used to build the _targetIDE_, use the _About_ dialog screen for the _targetIDE_.
+Next to **Build #** is the BRANCH.BUILD.FIX version of the IntelliJ Platform used to build the _targetIDE_.
+In the example shown below, the (BRANCH.BUILD.FIX) version of the IntelliJ Platform is `192.6817.19`.
+The product version is 2019.2.3.
+
+![Example PyCharm](img/pycharm_build.png){:width="600px"}
+
+If the product version isn't clear on the _About_ screen, consult the individual product pages in Part VIII.
+
+Using [Toolbox](https://www.jetbrains.com/toolbox-app/) is an easy way to find build numbers for recent versions.
+In the example below, IntelliJ IDEA Community (which defines the IntelliJ Platform) for v2019.2.3 is build number `192.6817.14`.
+
+![Toolbox Lookup](img/toolbox_lookup.png){:width="400px"}
+
+Although the FIX versions are different, this is not uncommon between products, and the builds are still compatible. 
+The BRANCH and BUILD numbers match, therefor in this PyCharm example: 
+* The _targetIDE_ (PyCharm) is `192.6817.19`, 
+* The _baseIntelliJPlatformVersion_ (IntelliJ IDEA Community Edition) is `192.6817.14`  
+
+### Configuring the Plugin build.gradle File
+For a project developing a plugin for _targetIDE_, configure the `build.gradle` file as follows:
```

Comment:
could we reformat this part, so that not nearly whole paragraph is indented?

Reply:
I reformatted to use paragraphs instead of the 3 major bullet items with sub-bullets.